### ISSUE TITLE
Allcoin.py: wrong method name which is supposed to be inherited from parent class

### ISSUE THREAD

**Issue by bugking** [ccxt/ccxt#849]
- Operating System:
- Language +version:  Python (both 2 & 3)
- CCXT version: latest
- Exchange:  all coin.com
- Method: 

### Description

in allcoin.py, there's a method named get_order_status, but it's never called. But in its parent class okcoinusd, there's a method parse_order_status, I believe it should be a typo in the allcoin.py since:

  * whenever fetching an cancelled order, the response contains a key 'status' with value as 10, which is supposed to be English 'cancelled'

please fix it ;)

### Code to reproduce the bug

```
COPYPASTE_YOUR_CODE_TO_REPRODUCE_THE_BUG_HERE
```

### Verbose output

[Troubleshooting](https://github.com/kroitor/ccxt/wiki/Manual#troubleshooting)

```
COPYPASTE_YOUR_VERBOSE_OUTPUT_HERE
```

**For questions and discussions join our gitter chat:** 

[![Gitter](https://badges.gitter.im/ccxt-dev/ccxt.svg)](https://gitter.im/ccxt-dev/ccxt?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)

**Comment by kroitor** [comment]
> please fix it ;)

Sure ) However,

> whenever fetching an cancelled order, the response contains a key 'status' with value as 10, which is supposed to be English 'cancelled'

The word will be English 'canceled' ) To us it looks like a more common spelling (as returned from exchanges). Let us know if you don't think so.

**Comment by kroitor** [comment]
@bugking can you please paste the `.verbose = true` output of that request?

**Comment by bugking** [comment]
sorry, maybe I didn't explain my issue very well:

  in the fetch_order method, it returns a dict like following [I have hacked the method a bit to avoid the following two issues caused by allcoin side, to make sure the method can work well]:

{'amount': 0.017,
 'average': 0.0,
 'cost': 0.0,
 'datetime': '2017-12-18T08:23:03.853Z',
 'fee': None,
 'filled': 0.0,
 'id': 52817935,
 'info': {'amount': 0.017,
  'avg_price': 0.0,
  'create_data': 1513585382853,
  'deal_amount': 0.0,
  'order_id': 52817935,
  'price': 19300.7,
  'status': 10,
  'symbol': 'btc_ck.usd',
  'type': 'buy'},
 'price': 19300.7,
 'remaining': 0.017,
 'side': 'buy',
 'status': 10,
 'symbol': 'BTC/CK.USD',
 'timestamp': 1513585382853,
 'type': 'limit'}

the verbose response output from allcoin is:

POST https://api.allcoin.com/api/v1/order_info 
Response: Cache-Control: no-cache
Content-Type: application/json; charset=utf-8
Date: Mon, 18 Dec 2017 08:26:54 GMT
Expires: -1
Pragma: no-cache
Server: Microsoft-IIS/8.5
X-AspNet-Version: 4.0.30319
X-Powered-By: ASP.NET
Content-Length: 306
Connection: Close

 {
  "result": true,
  "order": [
    {
      "amount": 0.0170,
      "deal_amount": 0.0000,
      "avg_price": 0.0000,
      "create_data": 1513585382853,
      "order_id": 52817935,
      "price": 19300.7000,
      "status": 10,
      "symbol": "btc_ck.usd",
      "type": "buy"
    }
  ]
}


apparently, there's a couple of issues, in both allcoin side & ccxt implementation:

1) the response is supposed to contain a key 'orders', instead of 'order', according to the API document [https://allcoin.com/About/APIReference/](https://allcoin.com/About/APIReference/). It's an issue from allcoin side;

2) there's a typo in the response as well: 'create_data' should be 'create_date' it's another issue from allcoin side;

3) let's take a look of the return value of fetch_order: it contains a key 'status', which is 10 in the example above.  I checked the source code, found 'status' value is supposed to be parsed as English in **parse_order_status** method ( okcoinusd.py line 359), but it's not contained the condition "status == '10'".
  but I do find a method **get_order_status** in allcoin.py (line 84), which comes from nowhere, and not called at all. that's why I believe that it should be a typo: the design is like this: parse_order_status is defined in parent class and can be overridden in every exchange class, to adapt definition of order_status in each exchange.

So the fix should be simple: rename the get_order_status as parse_order_status in allcoin.py
